docs/core.html

```diff
@@ -288,7 +288,7 @@ <h1 class="title">Module <code>dynamicio.core</code></h1>
                     if len(set([type(v) for v in df[column_name].values])) &gt; 1:  # pylint: disable=consider-using-set-comprehension
                         logger.warning(CASTING_WARNING_MSG.format(column_name, expected_dtype, found_dtype))  # pylint: disable=logging-format-interpolation
                         logger.info(NOTICE_MSG.format(column_name))  # pylint: disable=logging-format-interpolation
-                    df.loc[:, column_name] = df[column_name].astype(self.schema[column_name])
+                    df[column_name] = df[column_name].astype(self.schema[column_name])
```

Comment:
Just use`pd.assign` ([docs](https://pandas.pydata.org/docs/reference/api/pandas.DataFrame.assign.html))
You'll stop getting the warning and the behaviour isn't likely to change.

Reply:
> If you do not wish to maintain the original values, then clearly df["A"] = 1 will be more appropriate. This also explains the speed difference, by necessity .assign must copy the data while [...] does not. 

from https://stackoverflow.com/questions/48177914/why-use-pandas-assign-rather-than-simply-initialize-new-column

I don't want to copy anything.

Reply:
Fair enough